### ISSUE TITLE
Add to proposed tooling

### ISSUE THREAD

**Issue by arlemi** [postman-open-technologies/convert-postman-collections-to-openapi#1]
Work had been started on an official support for converting collections to OpenAPI 3, we might want to use that as a base, or contact @VShingala to see where the project is too!

Cf. https://github.com/postmanlabs/openapi-to-postman/pull/312

**Comment by VShingala** [comment]
Yeah, we had done significant work on this under [this branch](https://github.com/postmanlabs/openapi-to-postman/tree/feature/postman-to-openapi) of https://github.com/postmanlabs/openapi-to-postman repository.
